creator-node/src/routes/tracks.js

```diff
@@ -94,6 +94,27 @@ module.exports = function (app) {
     })
   )
 
+    /**
+     * Delete all temporary transcode artifacts from track transcode handoff flow.
+     * This is called on the node that was handed off the transcode to clear the state from disk
+     */
+  app.post(
+    '/clear_transcode_and_segment_artifacts',
```

Comment:
This can only be called by a valid SP for a folder inside tmp_track_artifacts, which need to be cleared anyway. On primaries this folder is deleted right after the transcoded files are copied into their permanent locations.

Also this is a safe operation because removeTrackFolder calls lstat before doing any fs operations, so if the dir doesn't exist it throws an error